src/serial_bridge_index.cpp

```diff
@@ -1862,3 +1866,43 @@ string serial_bridge::extract_utxos(const string &args_string) {
 
 	return ret_json_from_root(root);
 }
+
+string serial_bridge::verify_trezor_key_image(const string &args_string)
+{
+	boost::property_tree::ptree json_root;
```

Comment:
nit: what's with the indentation here? Are we mixing tabs and spaces?

Reply:
It looks normally in the editor. 

<img width="724" alt="Screenshot 2025-03-26 at 19 48 42" src="https://github.com/user-attachments/assets/544d7c75-198d-4f8a-bd37-1757f4cf2e78" />